docs/release-notes.md

```diff
@@ -19,6 +19,14 @@ You can also view the [Releases](https://github.com/n8n-io/n8n/releases){:target
 
 --8<-- "_snippets/update-n8n.md"
 
+## Semantic versioning in n8n
```

Comment:
Could we please align the description with the wording we use in the migration guide?